packages/react-icons/build/icons.js

```diff
@@ -1,74 +1,746 @@
 module.exports = {
-  fontAwesome: [
-    'angle-double-left',
-    'angle-double-right',
-    'angle-down',
-    'angle-left',
-    'angle-right',
-    'angle-up',
-    'arrow-circle-down',
-    'arrow-circle-up',
-    'balance-scale',
-    'ban',
-    'bell',
-    'bug',
-    'caret-down',
-    'caret-left',
-    'caret-right',
-    'caret-up',
-    'check-circle',
-    'check',
-    'clipboard-check',
-    'clock',
-    'columns',
-    'cube',
-    'cubes',
-    'ellipsis-v',
-    'exclamation-circle',
-    'exclamation-triangle',
-    'external-link-alt',
-    'external-link-square-alt',
-    'filter',
-    'flag',
-    'folder-open',
-    'folder',
-    'hdd',
-    'history',
-    'info-circle',
-    'key',
-    'list',
-    'lock-open',
-    'lock',
-    'map-marker',
-    'memory',
-    'minus',
-    'pause-circle',
-    'pencil-alt',
-    'plus-circle',
-    'plus-square',
-    'plus',
-    'print',
-    'question-circle',
-    'redo',
-    'search',
-    'sliders-h',
-    'sort-alpha-down',
-    'sort-alpha-up',
-    'sort-numeric-down',
-    'sort-numeric-up',
-    'sort',
-    'sync-alt',
-    'table',
-    'tachometer-alt',
-    'th-large',
-    'th',
-    'thumbtack',
-    'times-circle',
-    'times',
-    'trash',
-    'user-friends',
-    'user',
-    'window-restore',
-    'wrench'
-  ]
+  fontAwesome: {
+    solid: [
```

Comment:
it looks like `@fortawesome/free-solid-svg-icons` should have an export named `fas` that includes an object with all the icons.  You should be able to iterate through that to generate these.  Or really just `Object.keys(fas)` here
```js
const { fas } = require('@fortawesome/free-solid-svg-icons');
module.exports = {
  fontAwesome: Object.keys(fas),
};
```
You will probably also need to tweek `generateIcons` as well since i am added the `fa-` to them.  these will come preformatted to `faAddressBook`. We will need to remove the `fa` for the name, but it should be a small change.  This will make this more maintainable since this list could change at anytime.

Reply:
I like this approach! I should have looked into the package and not blindly just copy the names from FA page, I'll change it to work like this.